kuma/search/filters.py

```diff
@@ -84,13 +98,14 @@ class SearchQueryBackend(BaseFilterBackend):
     ]
 
     def filter_queryset(self, request, queryset, view):
-        search_param = request.QUERY_PARAMS.get(self.search_param, None)
+        qs = self.validate_query_params(request)
```

Comment:
technically that's not a `qs` (~query string) but query params, any chance to call that variable `validated_query_params` or `validated_params`?